addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -208,7 +205,11 @@ def _export_as_xml(self):
         :return: The XML content as str.
         '''
         template_values = self._prepare_fatturapa_export_values()
-        content = self.env.ref('l10n_it_edi.account_invoice_it_FatturaPA_export')._render(template_values)
+        if(not self.env['account.edi.format']._l10n_it_is_simplified_document_type(template_values['document_type'])):
+            content = self.env.ref('l10n_it_edi.account_invoice_it_FatturaPA_export')._render(template_values)
+        else:
+            content = self.env.ref('l10n_it_edi.account_invoice_it_simplified_FatturaPA_export')._render(template_values)
+            self.message_post(body='A simplified invoice was created instead of an ordinary one because the address of the customer is incomplete, it is an domestic invoice and the total amount is <= 400€.')
```

Comment:
Should be translated (and maybe avoid too long lines also, although too short is bad too)

Reply:
I agree with the message, but I would maybe remove the '<=' sign.. something like this? 

"A simplified invoice was created instead of an ordinary one. This is because the invoice is a domestic invoice with a total amount of less than or equal to 400€ and customer's address is incomplete."

(we will need translation terms anyway)